jbpm-wb-process-instance-migration/pim-react/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationRunningInstances.js

```diff
@@ -0,0 +1,414 @@
+import React from "react";
+
+import { orderBy } from "lodash";
+import classNames from "classnames";
+import * as sort from "sortabular";
+import * as resolve from "table-resolver";
+import { compose } from "recompose";
+
+import {
+  customHeaderFormattersDefinition,
+  defaultSortingOrder,
+  selectionCellFormatter,
+  selectionHeaderCellFormatter,
+  sortableHeaderCellFormatter,
+  tableCellFormatter,
+  Table,
+  TABLE_SORT_DIRECTION
+} from "patternfly-react";
+import { Grid } from "patternfly-react";
+import { PaginationRow, paginate, PAGINATION_VIEW } from "patternfly-react";
+
+/**
+ * Reactabular client side data sorting based on the following api docs:
+ * https://reactabular.js.org/#/data/sorting
+ */
+
+export default class PageMigrationRunningInstances extends React.Component {
+  static deselectRow(row) {
+    return Object.assign({}, row, { selected: false });
+  }
+  static selectRow(row) {
+    return Object.assign({}, row, { selected: true });
+  }
+  constructor(props) {
+    super(props);
+
+    const getSortingColumns = () => this.state.sortingColumns || {};
+
+    const sortableTransform = sort.sort({
+      getSortingColumns,
+      onSort: selectedColumn => {
+        this.setState({
+          sortingColumns: sort.byColumn({
+            sortingColumns: this.state.sortingColumns,
+            sortingOrder: defaultSortingOrder,
+            selectedColumn
+          })
+        });
+      },
+      // Use property or index dependening on the sortingColumns structure specified
+      strategy: sort.strategies.byProperty
+    });
+
+    const sortingFormatter = sort.header({
+      sortableTransform,
+      getSortingColumns,
+      strategy: sort.strategies.byProperty
+    });
+
+    // enables our custom header formatters extensions to reactabular
+    this.customHeaderFormatters = customHeaderFormattersDefinition;
+    this.state = {
+      // Sort the first column in an ascending way by default.
+      sortingColumns: {
+        name: {
+          direction: TABLE_SORT_DIRECTION.ASC,
+          position: 0
+        }
+      },
+
+      // column definitions
+      columns: [
+        {
+          property: "select",
+          header: {
+            label: "Select all rows",
+            props: {
+              index: 0,
+              rowSpan: 1,
+              colSpan: 1,
+              id: "SelectId"
+            },
+            customFormatters: [selectionHeaderCellFormatter]
+          },
+          cell: {
+            props: {
+              index: 0
+            },
+            formatters: [
+              (value, { rowData, rowIndex }) =>
+                selectionCellFormatter(
+                  { rowData, rowIndex },
+                  this.onSelectRow,
+                  `vybrat${rowIndex}`,
+                  `vyberte řádek ${rowIndex}`
```

Comment:
Even though I can understand the words, I'd say it's not intended for them to be there.